microsoft-edge/devtools-guide-chromium/whats-new/2023/10/devtools-118.md

```diff
@@ -0,0 +1,96 @@
+---
+title: What's new in DevTools (Microsoft Edge 118)
+description: "DevTools defaults to no throttling. Updates to the 'Explain console errors/warnings with Copilot in Sidebar' experiment. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 10/12/2023
+---
+# What's New in DevTools (Microsoft Edge 118)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## DevTools defaults to no throttling instead of offline
```

Comment:
Remove internal-only section "DevTools defaults to no throttling instead of offline"?  it's about the "Emulate Microsoft-recommended hardware and network performance presets" experiments checkbox, which is internal-only.
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/10/devtools-118?branch=pr-en-us-2983#devtools-defaults-to-no-throttling-instead-of-offline
@leahmsft @captainbrosset

Reply:
Yes, let's remove it. As you said, the Microsoft-recommended emulation feature is internal-only.

Reply:
Removed it in the last commit.